api/types/backend/backend.go

```diff
@@ -64,7 +65,7 @@ type LogSelector struct {
 // behavior of a backend.ContainerStats() call.
 type ContainerStatsConfig struct {
 	Stream    bool
-	OutStream io.Writer
+	OutStream http.ResponseWriter
```

Comment:
I ended up changing it due to https://github.com/moby/moby/blob/22472c8be50ca0ddaf8207d46c6a9f5e792bd0af/daemon/stats.go#L41-L47

(also a reason I opened this for discussion)